Kitodo/src/main/webapp/pages/inc_Prozessverwaltung/Prozesse_Liste.xhtml

```diff
@@ -375,14 +375,13 @@
             </h:link>
 
             <!-- Bearbeiten-Schaltknopf: Vorlagen -->
-            <h:commandLink action="#{ProzessverwaltungForm.editProcess}" id="action11"
-                           rendered="#{(LoginForm.maximaleBerechtigung == 1) and (ProzessverwaltungForm.modusAnzeige=='vorlagen')}"
-                           title="#{msgs.prozessBearbeiten}">
-                <h:graphicImage value="/pages/images/buttons/goInto.gif" alt="edit template"
+            <h:link outcome="/newpages/ProzessverwaltungBearbeiten"
+                    rendered="#{(LoginForm.maximaleBerechtigung == 1) and ProzessverwaltungForm.modusAnzeige=='vorlagen'}"
+                    title="#{msgs.prozessBearbeiten}">
+                <h:graphicImage value="/newpages/images/buttons/goInto.gif" alt="edit template"
```

Comment:
Everything is moved from `newpages` to `pages` directory, so please did not change path to this image back to `newpages`.